objects/obj_camera/Step_0.gml

```diff
@@ -11,8 +11,8 @@ if (instance_exists(follow)) {
 	y += (yTo - y) / readjust_speed;
 	
 	//center in room
-	x = clamp(x,view_w_half+buff,room_width - view_w_half-buff);
-	y = clamp(y,view_h_half+buff,room_height - view_h_half-buff);
+	//x = clamp(x,view_w_half+buff,room_width - view_w_half-buff);
```

Comment:
Can you add the comment //Turned off for proc gen
just like you did for the player create.